src/core/main.lua

```diff
@@ -143,6 +144,14 @@ end
 function shutdown (pid)
    if not _G.developer_debug and not lib.getenv("SNABB_SHM_KEEP") then
       shm.unlink("/"..pid)
+
+      -- Look through the named apps and unlink any which are for this process.
```

Comment:
Yes I think a backlink is the right thing here, please add one so that we can avoid this loop and the one in top.lua.